pom.xml

```diff
@@ -0,0 +1,73 @@
+ <!--
+
+    Copyright (C) 2021 Red Hat, Inc. (jdcasey@commonjava.org)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <parent>
+    <groupId>org.commonjava</groupId>
+    <artifactId>commonjava</artifactId>
+    <version>16</version>
+  </parent>
```

Comment:
Maybe we need a different parent pom here to consolidate common libs like quarkus.

Reply:
Do we use the different groupID for the micro services and event handlers ? I will update this after we lock that down.

Reply:
I think the "org.commonjava" for groupId is ok. For the artifact-id, how about "service-parent"? And I think we need a separate repo to hold this parent pom.

Reply:
"service-parent" looks good.